ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/OWNERS

```diff
@@ -9,6 +9,7 @@ approvers:
 - mffiedler
 - paigerube14
 - qiliredhat
+- svetsa-rh
```

Comment:
this file should probably also use the symbolic link

Reply:
Updated.